problems/arraysWithLoops.js

```diff
@@ -66,24 +111,38 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
+const findSecondSmallest = (nums) => {
+  let secondSmallest = Math.min.apply(null, nums);
+  nums.indexOf(secondSmallest), 1
+  return secondSmallest +1;
```

Comment:
This function won't actually work, it just happens to pass these speck Your secondSmallest should not be the smallest plus 1. 
Same goes for secondLargest below. Please revisit these and get them working for all cases. [1, 3, 5]